ArduCopter/AP_Arming.cpp

```diff
@@ -471,10 +471,10 @@ bool AP_Arming_Copter::arm_checks(bool display_failure, AP_Arming::Method method
         return false;
     }
 
-    // if we are using motor interlock switch and it's enabled, fail to arm
-    // skip check in Throw mode which takes control of the motor interlock
+    // if we are using Throttle Hold switch and it's engaged, fail to arm
+    // skip check in Throw mode which takes control of the Throttle Hold
     if (copter.ap.using_interlock && copter.ap.motor_interlock_switch) {
-        check_failed(ARMING_CHECK_NONE, display_failure, "Motor Interlock Enabled");
+        check_failed(ARMING_CHECK_NONE, display_failure, "Throttle Hold Engaged");
```

Comment:
In my implementation of this I used different warning message and made it separate for heli and multirotor
https://github.com/ChristopherOlson/ardupilot/commit/f22221ef4bfd0d875535e23ebcf10af82e70851b

Reply:
The next commit corrects the naming of those missing on AP_Arming, that I missed only first pass.

If the consensus comes to separate copter and multi rotor, then I can go and add the if's

As a question for consistency I've used Throttle Hold Engaged and Disengaged, where when THold is engaged the throttle is inactive, and THold disengaged the throttle is "free"

Reply:
That's the correct logic. It would require a lot of #if FRAME_CONFIG == HELI_FRAME statements to separate all this. Being there was a push to get rid of as many of those statements as possible in the code, probably wouldn't hurt to run it by Randy and tridge to see what they think. I don't think the multi pilots see the "motor interlock" message near as much as heli pilots because multi's don't have an actual throttle hold. They just arm and run, push the throttle and go. Helicopters are a lot more complicated with autorotation capability to reduce to ground idle power in flight via throttle hold. Throttle hold is used to engage the clutch for runup for pistons and turbines. It is used as a safety for electrics to prevent unwanted motor start. Throttle hold has a been a staple in RC helicopters for better than 40 years. It is just unfortunate that it had to be implemented in ArduPilot with reverse logic and a weird name that heli pilots don't even know what it means.